examples/sharded/README.md

```diff
@@ -0,0 +1,73 @@
+
+# Sharded example
+
+In this example, we use helm charts to deploy a sharded version of KCP.
+
+The final result from this example is a KCP cluster with 2 shards in 2 different
+namespaces, 1 frontend proxy and 1 cache server.
+
+### Prerequisites
+
+- Cert-manager installed in the cluster
+- Helm 3
+- reflector emberstack helm repository added for secret replication
+
+Namespace structure:
+```text
+kcp-certs - namespace for PKI infrastructure & certificates
+kcp-alpha - namespace for shard 1
+kcp-beta - namespace for shard 2
+kcp-proxy - namespace for frontend proxy
+kcp-cache - namespace for cache server
```

Comment:
How are certificate secrets working across namespaces? This might tie into my previous comment about where certificates should exist.

Reply:
For now I used reflector to sync them around.
But its up for users how they implement. They might not use cert-manager at all and do secrets their own way

Reply:
Ah sorry, that is already mentioned in the document. I skipped over it because I was asking myself this question before getting to this part of the PR and was looking for a location to open the discussion.